Objects/TI4_Helpers/TI4_UnitHelper.ttslua

```diff
@@ -1419,9 +1421,17 @@ function getColorToUnitModifiers()
 
     -- Add commanders (including Alliance and Imperia sharing).
     local colorToCommanders = _factionHelper.getColorToCommanders()
+    local toggleActiveCommanderToColor = {}
     for color, commanders in pairs(colorToCommanders) do
         for _, commander in ipairs(commanders) do
-            addModifier(color, commander)
```

Comment:
Perhaps it would be better to have toggle active on the alliance card?

Reply:
(can inject the modifiers based on unitModifier[faction.commander].toggleActive being set)

Reply:
That would be helpful. Though it leaves open the question of how Imperia factions would enable/disable a commander's combat effect. Can relatively large buttons be added to small tokens easily?